ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -23,16 +32,14 @@ controlPlane:
   platform:
     aws:
       zones:
-      - ${ZONE1:-${REGION}a}
-      - ${ZONE2:-${REGION}b}
```

Comment:
How about doing only this in both places to avoid other changes
`[[ "$ZONE" == "single" ]] && echo "- ${ZONE2:-${REGION}b}"`

Reply:
aws:
      zones:
      - ${ZONE1:-${REGION}a}
      `[[ "$ZONE" == "single" ]] && echo "- ${ZONE2:-${REGION}b}"`